File: 002/002.ts
const numbers: number[] = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
let sum: number = 0;

for (let i = 0; i < numbers.length; i++) {
  sum += numbers[i];
}

console.log("Sum:", sum);

File: 002/002.go
package main

import "fmt"

func main() {
    numbers := []int{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}
    sum := 0

    for _, num := range numbers {
        sum += num
    }

    fmt.Println("Sum:", sum)
}
